src/common/transformations/include/transformations/rt_info/disable_fp16_compression.hpp

```diff
@@ -25,14 +25,18 @@ TRANSFORMATIONS_API void do_not_postpone_fp16_compression(RTMap& rt_info);
 /**
  * @ingroup ie_runtime_attr_api
  * @brief DisableFP16Compression class represents runtime info attribute that marks operation
- * as prohibitted to convert to FP16 as part of Compressed Only format.
+ * as prohibited to convert to lower precision (e.g. to FP16) and they should be inferred precisely in the original precision.
  */
 class TRANSFORMATIONS_API DisableFP16Compression : public RuntimeAttribute {
 public:
-    OPENVINO_RTTI("disable_fp16_compression", "0");
+    OPENVINO_RTTI("precise", "0");
 
     DisableFP16Compression() = default;
 
+    bool visit_attributes(AttributeVisitor& visitor) override {
+        return true;
```

Comment:
Just for my education: does such implementation lead to missing `value` filed in IR as in the example in the description? Is it a "standard" way to implement Boolean attributes like this one?

Reply:
As far as i understood yes. E.g. existing `'decompression'` rt_info, i've seen the same thing
```
<layer id="4" name="bert/embeddings/word_embeddings" type="Convert" version="opset1">
    <data destination_type="f32" />
    <rt_info>
         <attribute name="decompression" version="0" />
    </rt_info>
```